scania-digital-design-system/tegel
# fix(modal): slot alignment

**Describe pull-request**  
Slot name convention 

**Solving issue**  
Fixes: DTS-1954

**How to test**  
1. Go to the preview link or branch out
2. Check if the names of the props are header, body, actions
3. Check if the props are working
4. Check if the header prop and/or slot are working
5. Check if "X" icons stays in place if no header text is provided

**Suggested test steps**
